llvm/lib/Target/AMDGPU/Disassembler/AMDGPUDisassembler.cpp

```diff
@@ -1254,6 +1255,20 @@ void AMDGPUDisassembler::convertVOPCDPPInst(MCInst &MI) const {
                          AMDGPU::OpName::src1_modifiers);
 }
 
+void AMDGPUDisassembler::convertVOPC64DPPInst(MCInst &MI) const {
+  unsigned Opc = MI.getOpcode();
+  unsigned DescNumOps = MCII->get(Opc).getNumOperands();
+
+  convertTrue16OpSel(MI);
+
+  if (MI.getNumOperands() < DescNumOps &&
+      AMDGPU::hasNamedOperand(Opc, AMDGPU::OpName::op_sel)) {
+    auto Mods = collectVOPModifiers(MI);
```

Comment:
Nit: it would help reading if we specify the type.
```suggestion
    VOPModifiers Mods = collectVOPModifiers(MI);
```

Reply:
done